KnockKnock-iOS/Repository/SNS/KakaoLoginManager.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  KakaoLoginManagerProtocol.swift
+//  KnockKnock-iOS
+//
+//  Created by Daye on 2022/12/23.
+//
+
+import Foundation
+
+import KakaoSDKUser
+
+protocol KakaoLoginManagerProtocol {
+
+
+}
+
+final class KakaoAccountManager: KakaoLoginManagerProtocol {
+
+  func loginWithKakao(completionHandler: @escaping (String) -> Void) {
```

Comment:
```suggestion
  func excute(completionHandler: @escaping (String) -> Void) {
```
이렇게 네이밍을 하는것이 좋겠습니다. 
`카카오 로그인 매니저` 클래스에서 `실행`을 한다로 의미하도록 말이죠
당연히 카카오 로그인 매니저니까 실행을하면 로그인을 하겠죠 
이렇게 하나의 기능만 정의가 되어있는 편이 좋겠습니다. SRP 를 잘 생각해주세요!